enderchest/enderchest.py

```diff
@@ -80,6 +82,7 @@ class EnderChest:
     _remotes: dict[str, ParseResult]
     offer_to_update_symlink_allowlist: bool = True
     sync_confirm_wait: bool | int = 5
+    do_not_sync = ["EnderChest/enderchest.cfg", "EnderChest/.*", ".DS_Store"]
```

Comment:
I tested this briefly on Windows, and it does appear that `fnmatch` is good with the "/," but the proof will be in the Windows CI run